app/views/ems_container/_main.html.haml

```diff
@@ -9,6 +9,8 @@
     -# commented out to hide Component Statuses table since the information is currently broken (see Issue #8572)
     -# = render :partial => "shared/summary/textual_multilabel", :locals => {:title => _("Component Statuses"),
     -#                                                           :items => textual_group_component_statuses}
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Meta-data"),
```

Comment:
Metadata is one word (though I would expect this to be Custom Attributes anyway just like everywhere else for consistency).